src/exercises/__init__.py

```diff
@@ -1,42 +1,86 @@
 from src.config import exercises_directory, current_working_exercises_directory
 
+from .course_representation_types import Directory, Exercice
```

Comment:
I would rename this file, what do you think about just `exercise.py`? Given this name, it would be relevant it contains an `Exercise` class and its container, the `Directory` file.